### PR TITLE
Long query

### PR DESCRIPTION
**Pull request by DuanKuanJun**

### CODE REVIEW

File: src/common/inc/tglobal.h [hunk 1]
```diff
@@ -221,6 +221,8 @@ extern uint32_t maxRange;
 extern uint32_t curRange;
 extern char Compressor[];
 #endif
+// long query 
+extern int8_t tsDeathLockKillQuery;
```
Comment (hjxilinx) [inline src/common/inc/tglobal.h]: deathlock? deadlock?

File: src/query/src/queryMain.c [hunk 2]
```diff
@@ -215,6 +215,51 @@ int32_t qCreateQueryInfo(void* tsdb, int32_t vgId, SQueryTableMsg* pQueryMsg, qi
...
+      if(unit_char == 'h'){
+        ms *= 3600*1000;
+      } else if(unit_char == 'm'){
+        ms *= 60*1000;
+      } else if(unit_char == 's'){
+        ms *= 1000;
+      }
+    }
+
+    printf("wait sleep %dms. sql=%s\n", ms, pQInfo->sql);
```
Comment (hjxilinx) [inline src/query/src/queryMain.c]: Please remove the printf, and use the log API instead.

File: src/common/inc/tglobal.h (hunk 1 above)
Comment (DuanKuanJun) [reply src/common/inc/tglobal.h]: modify to deadlock, done

**Review by hjxilinx — APPROVED** [review]

File: src/client/src/tscSystem.c [hunk 3]
```diff
@@ -59,7 +59,7 @@ void tscCheckDiskUsage(void *UNUSED_PARAM(para), void *UNUSED_PARAM(param)) {
   taosTmrReset(tscCheckDiskUsage, 20 * 1000, NULL, tscTmr, &tscCheckDiskUsageTmr);
 }
 
-void tscFreeRpcObj(void *param) {
+void tscFreeRpcObj(void *param, void* param1) {
```
Comment (hzcheng) [inline src/client/src/tscSystem.c]: Try your best to avoid changing codes in other modules.

**Review by hzcheng — APPROVED** [review]

**Comment by DuanKuanJun** [comment]
merge to develop

**Comment by DuanKuanJun** [comment]
reopen pr

**Comment by DuanKuanJun** [comment]
to develop

**Review by changxiaoqiang — APPROVED** [review]

**Pull request merged**